src/main/java/ua/ostapenko/hw10/PassengerCar.java

```diff
@@ -0,0 +1,17 @@
+package ua.ostapenko.hw10;
+
+public class PassengerCar extends Car {
+    private int passenger;
+
+    public PassengerCar(String series, int year, String color, int fuelLevel, int passenger) {
+        super(series, year, color, fuelLevel);
+        this.passenger = passenger;
+    }
+
+    public void addPassengers(int num) {
```

Comment:
is this method really adds passenger? 
looks like it set the number despite previous value.
It must prohibit adding passengers if it is not enough places for them

Reply:
addPassengers method is fixed